pkg/platform/packet/packet.go

```diff
@@ -614,6 +616,15 @@ func (c *config) validateOSVersion() hcl.Diagnostics {
 	return diagnostics
 }
 
+func (c *config) checkCPUManagerPolicy() hcl.Diagnostics {
+	wp := make(map[string]string)
+	for _, w := range c.WorkerPools {
+		wp[w.Name] = w.CPUManagerPolicy
+	}
```

Comment:
I wonder if we could extend `platform.WorkerPool` to make this simpler.

Reply:
I tried doing that but right now there are conflicts in the struct and the method names if I try to satisfy that interface.